galaxy/api/internal/views/combined.py

```diff
@@ -0,0 +1,101 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+from rest_framework import views
+from galaxy.main import models
+from rest_framework import response
+from galaxy.api.internal import serializers as internal_serializers
+from galaxy.api import serializers as v1_serializers
+
+
+class CombinedList(views.APIView):
+    def get(self, request):
+        # params: page, page size, namespace, name, description, type
+        # order by: name, downloads
+        namespace = request.GET['namespace']
+        page = int(request.GET.get('page', 1))
+        page_size = int(request.GET.get('page_size', 10))
+        type = request.GET.get('type', None)
```

Comment:
I would avoid use of 'type' as a local variable name if possible. It's kind of vague, but more importantly since it's also defined globally, it's very easy to accidentally not define it locally (since it _is_ defined) and end up getting weird behavior when comparing the identity of `<class 'type'>` with a string. 

Maybe 'package_type'? ( avoiding 'content_type' since that has another meaning already). Or 'type_' or '_type' perhaps.

Reply:
`_` suffix is normally used to avoid name collision with python built-ins. so `type_` is preferred.